clr-installer/main.go

```diff
@@ -123,122 +126,92 @@ func main() {
 
 		os.Exit(1)
 	}
-	_ = f.Close()
-}
 
```

Comment:
Please drop this extra blank line at the end of the function

Reply:
Thanks done!